flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/stream/StreamExecCalc.java

```diff
@@ -42,10 +44,13 @@
 @ExecNodeMetadata(
         name = "stream-exec-calc",
         version = 1,
+        producedTransformations = StreamExecCalc.SUBSTITUTE_TRANSFORMATION,
         minPlanVersion = FlinkVersion.v1_15,
         minStateVersion = FlinkVersion.v1_15)
 public class StreamExecCalc extends CommonExecCalc implements StreamExecNode<RowData> {
 
+    public static final String SUBSTITUTE_TRANSFORMATION = "substitute";
```

Comment:
again, let's call this differently. `substitute` makes no sense, it can be `calc`